src/main/java/frc/robot/ChallengeFour.java

```diff
@@ -90,13 +96,70 @@ public void UpdateTeleop() {
         /* 
         
             Explain your controls here, so the driver knows what to do. 
-                > Drive the robot using the left and right joysticks to control the speed of their respective wheels. 
+                > Drive the robot using the left and right joysticks to control the speed of their respective wheels OR left joystick only
+                > Rev up the shooter (big wheel) using left trigger and feed (blue wheel) using right trigger
+                > Increase/decrease shooter speed using right joystick
+                > Start intake using left bumper, reverse intake using right bumper
+                > Reset encoder using A button
         
         */
 
-        // This is a very basic way of driving using two joysticks. Think about other ways the robot can be driven. Which would be the easiest and/or most efficient for the driver?
-        controller.setDriveSpeed(xController.getY(Hand.kLeft), xController.getY(Hand.kRight));
+        // SmartDashboard values
+        SmartDashboard.putNumber("Distance travelled from start (ft)", encoderDistance);
+        SmartDashboard.putNumber("Shooter speed", shooterSpeed); 
 
+        // Reset distance recorded by encoder using A button
+        if (xController.getAButtonPressed()) {
+            controller.resetDistance();
+        }
+
+        // Tank drive (move each set of wheels forward/backward using left/right joysticks)
+        // controller.setDriveSpeed(leftStickY, rightStickY); [uncomment to use]
+        
+        // Arcade drive (move all wheels using left joystick)
+        // -- rotate left/right when leftStick moved left/right, go forward/backward when leftStick moved up/down
+        // -- add rotation to translation for both sets of wheels
+        // -- if leftStick is moved anywhere to left, right wheels should move faster than left wheels
+        // -- if leftStick is moved anywhere to right, left wheels should move faster than right wheels
+        // -- decrease leftStickX so that speed does not go out of bounds?
+        controller.setDriveSpeed(leftStickY + leftStickX * 0.2, leftStickY - leftStickX * 0.2);
+
+        // Use right joystick to increase/decrease launcher speed
+        if(rightStickY != 0) {
+            shooterSpeed += rightStickY;
```

Comment:
Is this intentional? Seems like the shooter speed would get added up every frame and would exceed (-1, 1) really fast. Maybe you wanted "=" not "+="?